README.md

```diff
@@ -46,6 +46,7 @@ Hence for security we have used both local storage and session storage, If users
     * [For UserConsentDetails](#userconsentdetails)
```

Comment:
@Bithika-Redapple You can separate your words for easier readability 

```
* [Recaptcha](#recaptcha)
* [Change Password](#change-password)
* [Change Email](#change-email)
* [Marketing Preferences](#marketing-preferences)
```

See what I did? I used title casing, and I also used syntax (lower casing) for tag naming conventions. These things are important for readability and following best practices.

Reply:
I have modified readme file.

Reply:
@Bithika-Redapple Great! Thank you. Feel free to merge when you are ready.